### ISSUE TITLE
Add a new makefile

### ISSUE THREAD

**Issue by Matthew-Tate-Scarbrough** [olivertaylor/Textplay#41]
The current makefile lists:

`install: textplay`
	`ln -s $(PWD)/textplay /usr/local/bin/`

What this does, is merely create a symbolic link inside of /usr/local/bin to a non-existent file, that, for some reason, is located in in root along with /etc, /home, /usr, etc. this would require the user to manually copy the actual program to /. I am unsure of it is required to work this way in the original version of OS X this program was written for.

It would be nice to see a makefile added for Linux, perhaps in a subfolder, called "Linux Install" or something. This is my suggestion:

`PREFIX = /usr/local`

`install: textplay`
`	mkdir -p $(DESTDIR)$(PREFIX)/bin`
`	cp -f textplay $(DESTDIR)$(PREFIX)/bin`
`	chmod 775 $(DESTDIR)$(PREFIX)/bin/textplay`

`uninstall: textplay`
`	rm -f $(DESTDIR)$(PREFIX)/bin/textplay`

Basically, if one run `make install`, this will, first, check to see if `/usr/local/bin` exists, if so, it will skip making it, if not, it will make it; second, it will check to see if `/usr/local/bin/textplay` exists, if so, it will first remove it, then copy the new one in, if not, then it will merely copy; and thirdly, it will change the user permissions to allow the user to be able to execute the command without the need of `sudo`.

**Comment by olivertaylor** [comment]
Yes, I've only really tested this on macOS.

Something like what you've suggested is probably a good idea, though I'm not really qualified to make this for Linux and would have no way of testing it.

The current makefile is designed to allow you to keep this cloned git repo anywhere you want (which is what the `$(pwd)` is for) and create a symlink somewhere on your $PATH. This way you can pull updates from this repo and the symlink would always point to the latest version.

Admittedly, this approach probably doesn't work everywhere.
